sdks/java/extensions/sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -160,7 +160,7 @@ SqlCreate SqlCreateExternalTable() :
 }
 {
 
-    <CREATE> <EXTERNAL> <TABLE> {
+    <EXTERNAL> <TABLE> {
```

Comment:
Does this mean a change in the SQL syntax (the removal of the  `create` ) ? If so; what are the backwards compatibility effects for user applications?

Reply:
Yes, CREATE is now implicit.